frontend/src/data/sorting/DatasetOrganizer.ts

```diff
@@ -0,0 +1,38 @@
+import { Breakdowns } from "../query/Breakdowns";
+import { ALL, UNKNOWN, UNKNOWN_HL } from "../utils/Constants";
+import { Row } from "../utils/DatasetTypes";
+import { AbstractDataSorter } from "./AbstractDataSorter";
+import { AgeSorter } from "./AgeSorter";
+import { AlphabeticalSorter } from "./AlphabeticalSorter";
+
+export class DatasetOrganizer {
```

Comment:
Yeah I was gonna keep the replacing data, but because the individual providers wrap the data with the response, moving this to the variable provider cant rewrite the data var since its readonly.  Was gonna address this maybe when I refactor the providers to move the wrapping with the response logic to the base class